utils/text_formatting.tsx

```diff
@@ -2,16 +2,22 @@
 // See LICENSE.txt for license information.
 
 import XRegExp from 'xregexp';
+
+// @ts-ignore TODO! ignore this for now because of the missing type definition
```

Comment:
Which type definitions are missing? If they're not too big, perhaps we could define them ourselves

Reply:
There's an existing PR for TS types on `mattermost-redux`'s here https://github.com/mattermost/mattermost-redux/pull/936. I think it is being reviewed.